src/index.html

```diff
@@ -11,6 +11,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    
+    <form 
```

Comment:
<img width="1049" alt="Screenshot 2023-01-13 at 15 40 38" src="https://user-images.githubusercontent.com/53228259/212333384-e6a0fb78-490c-4187-9587-3e15260ff56d.png">

general: fix all tags that looks like form tag